project/SparkBuild.scala

```diff
@@ -257,6 +257,7 @@ object SparkBuild extends PomBuild {
 
   val noLintOnCompile = sys.env.contains("NOLINT_ON_COMPILE") &&
       !sys.env.get("NOLINT_ON_COMPILE").contains("false")
+  lazy val skipLocalM2 = sys.env.getOrElse("SKIP_LOCAL_M2", "false").toBoolean
```

Comment:
We always need to check this status, right? So I don't think it needs to be marked as `lazy`.

Reply:
Updated.